src/femtomes.cpp

```diff
@@ -434,3 +639,101 @@ GPSDriverFemto::calculateBlockCRC32(uint32_t length, uint8_t *buffer, uint32_t c
 	return (crc);
 }
 
+void GPSDriverFemto::activateCorrectionOutput()
+{
+    if (_correction_output_activated || _output_mode != OutputMode::RTCM)
```

Comment:
You shouldn't have to check for `_correction_output_activated`, as this is called once during configure.
```suggestion
    if (_output_mode != OutputMode::RTCM)
```